api/src/test/java/org/openmrs/module/openconceptlab/client/OclClientTest.java

```diff
@@ -132,4 +141,43 @@ public void parseDateFromPath_shouldParseDate() throws Exception {
 		
 		assertThat(date, is(calendar.getTime()));
 	}
+
+	@Test
+	public void getOclReleaseVersion_shouldGetOclReleaseVersion() throws IOException {
+		String subscriptionToken = "53fc72f0498a707a26e4d903c0f24c2db24d1e35";
+		String version = oclClient.getOclReleaseVersion(URL_WITH_VERSION, subscriptionToken);
+		assertThat(version, is("v1.0"));
+	}
+
+	@Test
+	public void getExportUrl_shouldConstructCorrectExportUrlWhenVersionIsPassedToUrl() throws URIException {
+		String collectionVersion = "v1.0";
+		GetMethod getMethodUrlWithVersion = oclClient.getExportUrl(URL_WITH_VERSION, collectionVersion);
+		assertThat(getMethodUrlWithVersion.getURI().toString(), is(URL_WITH_VERSION + "/export"));
+	}
+
+	@Test
+	public void getExportUrl_shouldConstructCorrectExportUrlWhenVersionIsNotPassedToUrl() throws URIException {
+		String collectionVersion = "v1.0";
+		GetMethod getMethodUrlWithoutVersion = oclClient.getExportUrl(URL_WITHOUT_VERSION, collectionVersion);
+		assertThat(getMethodUrlWithoutVersion.getURI().toString(), is(URL_WITHOUT_VERSION + "/v1.0" + "/export"));
+	}
+
+	@Test
+	public void getExportUrl_shouldConstructCorretExportUrlWhenVersionIsNotPassedToUrl() throws URIException {
+		String collectionVersion = "v2.0";
+		GetMethod getMethod = oclClient.getExportUrl(URL_1, collectionVersion);
+		assertThat(getMethod.getURI().toString(), is(URL_1 + "v2.0/export"));
+	}
+
+	@Test
+	public void removeLastUrlForwardSlashIfExist_shouldRemoveLastForwardSlashIfItExist() {
+		String newUrl1 = oclClient.removeLastUrlForwardSlashIfExist(URL_1);
+		int Url1Count = StringUtils.countMatches(URL_1, "/");
+
+		int newUrl1Count = StringUtils.countMatches(newUrl1, "/");
+
+		assertNotEquals(Url1Count, newUrl1Count);
+		assertThat(Url1Count, greaterThan(newUrl1Count));

```

Comment:
Do we need these two assertions? In other words, if one is greater than the other, is there any possibility of being equal? 😊